aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -56,18 +58,49 @@ public static DescribeDbClustersRequest describeDbClustersRequest(final Resource
                 .build();
     }
 
+    public static DescribeDbClustersRequest describeDbClusterRequest(final String dbClusterIdentifier) {
+        return DescribeDbClustersRequest.builder()
+                .dbClusterIdentifier(dbClusterIdentifier)
+                .build();
+    }
+
     public static DescribeDbInstancesRequest describeDbInstancesRequest(final String nextToken) {
         return DescribeDbInstancesRequest.builder()
                 .marker(nextToken)
                 .build();
     }
 
+    public static DescribeDbInstancesRequest describeDbInstanceByDBInstanceIdentifierRequest(final String dbInstanceIdentifier) {
+        return DescribeDbInstancesRequest.builder()
+                .dbInstanceIdentifier(dbInstanceIdentifier)
+                .build();
+    }
+
+    public static DescribeDbInstancesRequest describeDbInstanceByResourceIdRequest(final String dbInstanceIdentifier) {
+        return DescribeDbInstancesRequest.builder()
+                .filters(software.amazon.awssdk.services.rds.model.Filter.builder()
+                        .name("dbi-resource-id").values(dbInstanceIdentifier).build())
```

Comment:
I just wanted to confirm if this filter is case insensitive. In case the customer specifies a mixed case for the `DbiResourceId` and we send it as is to the Webservice, will the target DbInstance be returned correctly? This is because the Webservice always converts all letters to lowercase for identifiers.

Reply:
The filter is not case sensitive, customers are free to use whichever case they prefer.